src/selectivity-single.js

```diff
@@ -216,6 +222,22 @@ var callSuper = Selectivity.inherits(SingleSelectivity, {
     /**
      * @private
      */
+    _mouseover: function() {
+
+      this.$('.selectivity-single-select').toggleClass("hover", true);
+    },
+
+    /**
+     * @private
+     */
+    _mouseout: function() {
+
+      this.$('.selectivity-single-select').toggleClass("hover", false);
```

Comment:
It seems that for both the single and the multiple implementations, you're using the top-level element to add the class to. So wouldn't it make sense to implement this functionality in the base class using `this.$el.children().toggleClass(...)`?